src/main/java/org/codehaus/mojo/sql/SqlExecMojo.java

```diff
@@ -378,7 +378,7 @@ public class SqlExecMojo extends AbstractMojo {
      * @since 1.4
      */
     @Parameter(defaultValue = ",")
-    private String outputDelimiter;
+    private String outputDelimiter = ",";
```

Comment:
According to https://maven.apache.org/guides/plugin/guide-java-plugin-development.html, the default value definitions should not put value to the variable.

Reply:
Fixed